legal-api/report-templates/certificateOfAmalgamation.html

```diff
@@ -34,19 +34,28 @@
                         <tr class="no-page-break">
                             <td class="col-50">
                                 <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                    and {{ entity.name }} ({{ entity.identifier }})
+                                    {% if entity.legalName %}
+                                        and {{ entity.legalName }} ({{ entity.identifier }})
+                                    {%else%}
+                                        and {{ entity.name }} ({{ entity.identifier }})
+                                    {%endif%}
                                 </div>
```

Comment:
Essentially, foreign companies will have 'legalName' instead of 'name' in the json (and possibly vice versa), this is to handle those situations.